docs/book/v2/hot-code-reload.md

```diff
@@ -40,6 +40,14 @@ return [
 ];
 ```
 
+## Logging
+
+When a file is reloaded, a *notice* line will be logged with the message
+`File changed {path}, reloading`.
```

Comment:
Could we maybe change this to: `Reloading due to file change: {path}`? That puts the action ("reloading") first in the message, and the variable information (path) last, which will be easier to parse using features such as grep.

Reply:
I think it's a great idea.